index.py

```diff
@@ -682,8 +682,27 @@ def deg2num(lat_deg, lon_deg, zoom):
 
 
 if __name__ == "__main__":
-    web.wsgi.runwsgi = lambda func, addr=None: web.wsgi.runfcgi(func, addr)
+    # standalone run
+
+    # stop listening on 0.0.0.0
+    if not sys.argv[1:]:
+        sys.argv.append('127.0.0.1')
```

Comment:
I not sure that understand this comment completely do we have issue with `0.0.0.0` host now?
Why we need change it?
Can we avoid `sys.argv` patching there?

Reply:
`0.0.0.0` means that the web app in development mode will be accessible to local network, and I don't trust `index.py` until I review it. I guess I am not alone, so I prefer to stick to good security practices.

No, it is impossible to set the host to listen to in `web.py` without patching `sys.argv` https://github.com/webpy/webpy/issues/639

Reply:
@abitrolly , you can pass `argv` when run `index.py` and solve your issue in case if you have any concerns to run in on `0.0.0.0`, I like idea to have `0.0.0.0` host that helps test view on mobile devices for example. I don't have strong opinion about security weakness using `0.0.0.0` so I recommend to split this changes to separate PR.

Reply:
Fair enough. I will remove that part. Probably not today.